app/main.py

```diff
@@ -1,25 +1,109 @@
-from abc import ABC
+from abc import ABC, abstractmethod
 
 
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.status_access = None
+
+    def __set_name__(self,
+                     owner: "SlideLimitationValidator",
```

Comment:
```suggestion
                     owner: Type[SlideLimitationValidator],
```

Reply:
change here and in all of code

Reply:
owner: Type[SlideLimitationValidator],
E   NameError: name 'SlideLimitationValidator' is not defined

Reply:
If I arrange the classes by spelling, it also does not work because the class is not created for typing, and if I rearrange it, it comes out in other classes.
I can write "str" instead of a string constant.